sdk/nodejs/tests/output.spec.ts

```diff
@@ -632,29 +623,6 @@ describe("output", () => {
 
             assert.strictEqual(await r.isKnown, false);
             assert.strictEqual(await r.isSecret, true);
-            assert.strictEqual(await r.promise(), "inner");
-        });
-
-        it("propagates secret on unknown known output", async () => {
```

Comment:
This feels a bit unintuitive, but these tests don't make sense if we are allowing unknown outputs.  There will not be any value, so there's nothing to be secret.